core/src/worker/activities/activity_heartbeat_manager.rs

```diff
@@ -141,9 +142,10 @@ impl ActivityHeartbeatManager {
                             };
                             }
                             HeartbeatExecutorAction::Report { task_token: tt, details } => {
-                                match sg
+                                dbg!("reporting", &details);
```

Comment:
Some `dbg!` were left, here and other places.

Reply:
Yeah that's on purpose for the server people to investigate the linked bug. This test is currently skipped.